spec/storage_v0_openapi.json

```diff
@@ -708,6 +730,118 @@
         }
       }
     },
+    "/object/upload/sign/{bucketName}/{wildcard}": {
+      "post": {
+        "summary": "Generate a presigned url to upload an object",
+        "tags": ["object"],
+        "parameters": [
+          {
+            "schema": { "type": "string", "example": "avatars" },
+            "example": "avatars",
+            "in": "path",
+            "name": "bucketName",
+            "required": true
+          },
+          {
+            "schema": { "type": "string", "example": "folder/cat.png" },
+            "example": "folder/cat.png",
+            "in": "path",
+            "name": "*",
+            "required": true
+          },
+          {
+            "in": "header",
+            "name": "authorization",
+            "required": true,
+            "schema": { "type": "string" },
+            "example": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24ifQ.625_WdcF3KHqz5amU0x2X5WWHP-OEs_4qj0ssLNHzTs"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful response",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "description": "Successful response",
+                  "type": "object",
+                  "properties": {
+                    "url": {
+                      "type": "string",
+                      "example": "/object/sign/upload/avatars/folder/cat.png?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1cmwiOiJhdmF0YXJzL2ZvbGRlci9jYXQucG5nIiwiaWF0IjoxNjE3NzI2MjczLCJleHAiOjE2MTc3MjcyNzN9.s7Gt8ME80iREVxPhH01ZNv8oUn4XtaWsmiQ5csiUHn4"
+                    }
+                  },
+                  "required": ["url"]
+                }
+              }
+            }
+          },
+          "4XX": {
+            "description": "Error response",
+            "content": {
+              "application/json": {
+                "schema": { "description": "Error response", "$ref": "#/components/schemas/def-1" }
+              }
+            }
+          }
+        }
+      },
+      "put": {
+        "summary": "Uploads an object via a presigned URL",
```

Comment:
curious about the params here in supabase-js.   Are these not relevant for swift?  Or just not handled yet?   Want to make sure I understand! 


<img width="527" alt="CleanShot 2023-04-03 at 09 30 42@2x" src="https://user-images.githubusercontent.com/105593/229503561-b51463a3-289a-4b83-8f8c-3e9a4de7d0f8.png">

Reply:
@saltcod got what you meant. I was working of Flutter V1 and JS V1 docs I think so the parameter details weren't there in those. I'll add them in the next version.